packages/grid/x-data-grid/src/components/containers/GridRootStyles.ts

```diff
@@ -237,11 +237,40 @@ export const GridRootStyles = styled('div', {
       justifyContent: 'center',
       color: borderColor,
     },
-    [`& .${gridClasses.columnHeaders}:hover .${gridClasses.columnSeparator}, .${gridClasses['columnSeparator--resizing']}`]:
-      {
-        visibility: 'visible',
-        width: 'auto',
+    '@media (pointer: fine)': {
```

Comment:
Probably doesn't matter too much for our use-case, but we should probably use `@media (hover: hover)` instead.

See the table at the bottom of this section: https://www.smashingmagazine.com/2022/03/guide-hover-pointer-media-queries/#combining-media-queries

Reply:
Thanks, that would be more accurate